packages/fluentui/accessibility/src/behaviors/Pill/pillsBehavior.ts

```diff
@@ -0,0 +1,19 @@
+import { Accessibility } from '../../types';
+import { FocusZoneDirection } from '../../focusZone/types';
+
+export const pillsBehavior: Accessibility<PillsBehaviorProps> = () => ({
+  attributes: {
+    root: {
+      role: 'listbox',
+      tabIndex: -1,
```

Comment:
Not entirely sure about the history here, but I think that WAI listbox standardn states that `aria-activedescendant` should be used to convey which is the active item, and the entire listbox is supposed to be focusable... did we find issues for screen readers with the WAI standard ? 

For reference: https://www.w3.org/TR/2017/WD-wai-aria-practices-1.1-20170628/examples/listbox/listbox.html

Reply:
I based the implementation on the `selectableListBehavior` but probably @kolaps33 can give us more insight here.

Reply:
> The listbox role supports the aria-activedescendant property, which provides an alternative to moving DOM focus among option elements when implementing keyboard navigation. For details, see Managing Focus in Composites Using aria-activedescendant.

activedescendant is supported, but rowing tabindex is always an option